### DISCUSSION TITLE
Open Links in External Apps

### DISCUSSION THREAD

**Discussion by blogmangit** [bromite/bromite#1471] [Q&A]
Is it possible at present to open links in external apps like YouTube links in YouTube app or newpipe or say reddit link's in reddit app?

**Comment by uazo** [comment]
> YouTube links in YouTube app or newpipe

no, for youtube it is currently wired and cannot be changed by the user

> say reddit link's in reddit app

for all other applications, there is a flag for that `chrome://flags/#disable-external-intent-requests`
by default it is active

**Reply by blogmangit** [reply]
Thanks a lot man I was looking for it.🤙👍

**Reply by blogmangit** [reply]
Hey man tried your suggested flag I mean disabled it but no change links still open in bromite instead off respective apps.
